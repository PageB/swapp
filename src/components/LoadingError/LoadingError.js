import React from 'react';

import Image from './../../assets/images/error.gif';

import styles from './LoadingError.module.scss';

const LoadingError = () => {
  return (
    <div className={styles.LoadingError}>
      <img className={styles.LoadingError__Image} src={Image} alt="Error" />
    </div>
  );
};

export default LoadingError;
