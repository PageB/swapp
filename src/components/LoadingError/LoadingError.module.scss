.LoadingError {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;

  &__Image {
    height: auto;
    width: 200px;
  }
}