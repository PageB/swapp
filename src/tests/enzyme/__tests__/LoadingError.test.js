import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import LoadingError from '../../../components/LoadingError/LoadingError';

configure({ adapter: new Adapter() });

describe('LoadingError Component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<LoadingError />);

    expect(wrapper).toMatchSnapshot();
  });
});
