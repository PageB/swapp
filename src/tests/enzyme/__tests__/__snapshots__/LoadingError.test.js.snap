// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingError Component should match snapshot 1`] = `
<div
  className="LoadingError"
>
  <img
    alt="Error"
    className="LoadingError__Image"
    src="error.gif"
  />
</div>
`;
